tests/ci_build/Dockerfile.gpu_mklml

```diff
@@ -15,4 +15,4 @@ RUN /install/ubuntu_install_scala.sh
 RUN wget --no-check-certificate -O /tmp/mklml.tgz https://github.com/01org/mkl-dnn/releases/download/v0.12/mklml_lnx_2018.0.1.20171227.tgz
 RUN tar -zxvf /tmp/mklml.tgz && cp -rf mklml_*/* /usr/local/ && rm -rf mklml_*
 
-ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib:/usr/lib/x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/5/
```

Comment:
I think most of these are by default and not needed, could you check the output of:

cat /etc/ld.so.conf.d/*

?

Why add them if not needed?

```
(mxnet_py3) ubuntu@ip-172-31-36-119:~/devel/mxnet$ docker run -ti nvidia/cuda:8.0-cudnn5-devel
root@741913f34dea:/# cat /etc/ld.so.conf.d/*
/usr/local/cuda-8.0/targets/x86_64-linux/lib
# libc default configuration
/usr/local/lib
/usr/local/nvidia/lib
/usr/local/nvidia/lib64
# Multiarch support
/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu
```

Reply:
I would remove this if not needed, it's confusing.